charts/merlin/templates/_helpers.tpl

```diff
@@ -57,3 +57,12 @@
 {{- printf .Values.mlflow.backendStoreUri -}}
 {{- end -}}
 {{- end -}}
+
+
+{{- define "merlin.getEnvironmentSecret" -}}
+{{- if .Values.mlp.environmentConfigSecret.name }}
+secretName: {{ .Values.mlp.environmentConfigSecret.name }}
+{{- else }}
+secretName: {{ template "merlin.fullname" .}}-environments
+{{- end -}}
+{{- end -}}
```

Comment:
I think it's better to just return the value without the `secretName: ` field. You can use it as a variable and no need to configure the indentation.